cmd/project.go

```diff
@@ -804,6 +806,81 @@ Examples:
 	},
 }
 
+var projectRollbackDeploymentCmd = &cobra.Command{
+	Use:   "rollback-deployment",
+	Short: "Rollback a deployment for a project",
+	Long: `Rollback a specific deployment for a project by selecting a project and deployment.
+
+Examples:
+  agentuity project rollback-deployment
+`,
+	Args: cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := env.NewLogger(cmd)
+		ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
+		defer cancel()
+		apikey, _ := util.EnsureLoggedIn(ctx, logger, cmd)
+		apiUrl, _, _ := util.GetURLs(logger)
+
+		selected := selectProject(ctx, logger, apiUrl, apikey, "Select a project to rollback a deployment")
+		if selected == "" {
+			return
+		}
+		selectedDeployment := selectDeployment(ctx, logger, apiUrl, apikey, selected, "Select a deployment to rollback")
+		if selectedDeployment == "" {
+			tui.ShowWarning("no deployment selected")
+			return
+		}
+		err := project.RollbackDeployment(ctx, logger, apiUrl, apikey, selected, selectedDeployment)
+		if err != nil {
+			tui.ShowError("%s", err.Error())
+			return
+		}
+		tui.ShowSuccess("Deployment rolled back successfully")
+	},
+}
+
+var projectDeleteDeploymentCmd = &cobra.Command{
+	Use:   "delete-deployment",
+	Short: "Delete a deployment for a project",
+	Long: `Delete a specific deployment for a project by selecting a project and deployment.
+
+Examples:
+  agentuity project delete-deployment
+`,
+	Args: cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := env.NewLogger(cmd)
+		ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
+		defer cancel()
+		apikey, _ := util.EnsureLoggedIn(ctx, logger, cmd)
+		apiUrl, _, _ := util.GetURLs(logger)
+
+		selected := selectProject(ctx, logger, apiUrl, apikey, "Select a project to delete a deployment")
+		if selected == "" {
+			return
+		}
+		selectedDeployment := selectDeployment(ctx, logger, apiUrl, apikey, selected, "Select a deployment to delete")
+		if selectedDeployment == "" {
+			tui.ShowWarning("no deployment selected")
+			return
+		}
+
+		if !tui.Ask(logger, "Are you sure you want to delete the selected deployment?", true) {
```

Comment:
use this above too

Reply:
both of these need a non TTY version flag like --force if you dont have tty so you can automate with scripts